Dockerfile.dapper

```diff
@@ -1,5 +1,5 @@
-FROM ubuntu:20.04
 # FROM arm=armhf/ubuntu:20.04 arm64=arm64v8/ubuntu:20.04
```

Comment:
Might as well remove this commented-out line as it doesn't seem we're going back to ubuntu following this change.

Reply:
Fixed in latest patchset.